chia/full_node/mempool_manager.py

```diff
@@ -504,6 +510,11 @@ async def validate_spend_bundle(
 
         timelocks: TimelockConditions = compute_assert_height(removal_record_dict, npc_result.conds)
 
+        if timelocks.assert_before_height is not None and timelocks.assert_before_height <= timelocks.assert_height:
+            return Err.IMPOSSIBLE_HEIGHT_ABSOLUTE_CONSTRAINTS, None, []  # MempoolInclusionStatus.PENDING
+        if timelocks.assert_before_seconds is not None and timelocks.assert_before_seconds <= timelocks.assert_seconds:
+            return Err.IMPOSSIBLE_SECONDS_ABSOLUTE_CONSTRAINTS, None, []  # MempoolInclusionStatus.PENDING
+
```

Comment:
These lead to `MempoolInclusionStatus.FAILED` not `MempoolInclusionStatus.PENDING`. Maybe this is a chance to add a comment on top of them to further clarify the behavior.